packages/web-components/fast-foundation/src/accordion-item/accordion-item.template.ts

```diff
@@ -15,10 +15,7 @@ export const accordionItemTemplate: (
     context: ElementDefinitionContext,
     definition: AccordionItemOptions
 ) => html`
-    <template
-        class="${x => (x.expanded ? "expanded" : "")}"
-        slot="item"
```

Comment:
Turns out FAST was automatically adding the slot under the hood. I assume only a tiny portion of the consumers will have to keep their eyes open for a breaking change, as long as they use a component extending `AccordionItem`